SeaPkg/Library/BasePeCoffLibNegative/BasePeCoffLibNegative.c

```diff
@@ -981,6 +981,21 @@ PeCoffImageDiffValidation (
           NextImageValidationEntryHdr = (IMAGE_VALIDATION_ENTRY_HEADER *)(ImageValidationEntrySelfRef + 1);
         }
 
+        break;
+      case IMAGE_VALIDATION_ENTRY_TYPE_POINTER:
+        if (ImageValidationEntryHdr->Size >= sizeof (EFI_PHYSICAL_ADDRESS)) {

```

Comment:
```suggestion
        if (ImageValidationEntryHdr->Size >= sizeof (UINTN)) {
```

`EFI_PHYSICAL_ADDRESS` is a `UINT64`. I think this will scale better.